Documentation/gettingstarted/minikube.rst

```diff
@@ -187,9 +187,9 @@ From the perspective of *deathstar* service providers only the ships with label
 
 .. parsed-literal::
 
-    $ kubectl exec xwing-cc65988f5-7cvn8 -- curl -s -XPOST 10.109.254.198/v1/request-landing
+    $ kubectl exec xwing-cc65988f5-7cvn8 -- curl -s -XPOST deathstar.default.svc.cluster.local/v1/request-landing
```

Comment:
Does just referring to it as `deathstar` not work?

Reply:
no it isn't working in the minikube. I checked the previous demo app and that was referring to the service the same way.